raiden/ui/app.py

```diff
@@ -426,6 +409,29 @@ def run_app(
         except AddressWrongContract:
             handle_contract_wrong_address('user_deposit', user_deposit_contract_address)
 
+    service_registry = None
+    if environment_type == Environment.DEVELOPMENT:
+        try:
```

Comment:
Same question as above. When basic routing is used we don't need to perform the check IMO.